Donation-platform/src/main/java/com/google/FindTrending.java

```diff
@@ -0,0 +1,85 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.lang.Double;
+import java.util.Set;
+import java.util.HashSet;
+import java.util.HashMap;
+
+public final class FindTrending {
```

Comment:
Rename to FindTrendingCharity?

Reply:
I think the name is good but if you were to change it I'd call it FindTrendingCharities to reflect the fact that query() returns a collection a charities.

Reply:
sure.

Reply:
I don't see the name changed?

Reply:
I changed the name and committed yesterday.

Reply:
weird, i still see the old name.